.github/workflows/static-analysis.yml

```diff
@@ -20,31 +20,8 @@ jobs:
       - name: Lint with flake8
         run: |
           flake8 --max-line-length=120 --import-order-style=pycharm --statistics \
-              --application-import-names asf_tools ArcGIS-toolbox/ASF_Tools.pyt asf_tools
+              --application-import-names asf_tools ArcGIS-toolbox/ASF_Tools.pyt src/asf_tools
 
 
   call-secrets-analysis-workflow:
-    uses: ASFHyP3/actions/.github/workflows/reusable-secrets-analysis.yml@v0.4.0
-
-
-  trufflehog:
```

Comment:
Are we removing our trufflehog static analysis, or has this been moved elsewhere?